codemod/base.py

```diff
@@ -26,6 +26,7 @@
 import sys
 import textwrap
 from math import ceil
+import tty, termios
```

Comment:
Perhaps you could use the `platform` module to only enable this feature on platforms that can support it? As you mentioned, we'd definitely want macOS support before merging.